### PR TITLE
Add PullRequest mergeable field.

### PR DESCRIPTION
**Pull request by calavera**
This field can have three states:

1. `null`: unknown state. Github has not calculated yet if it can be merged or not.
2. `true`: :+1: to merge.
3. `false`: :-1: to merge.

### CODE REVIEW

File: octokit/pull_requests.go [hunk 1]
```diff
@@ -91,6 +91,7 @@ type PullRequest struct {
 	Additions         int               `json:"additions,omitempty"`
 	Deletions         int               `json:"deletions,omitempty"`
 	ChangedFiles      int               `json:"changed_files,omitempty"`
+	Mergeable         *bool             `json:"mergeable,omitempty"`
```
Comment (pengwynn) [inline octokit/pull_requests.go]: There's also `mergeable_state` which returns one of "clean", "unstable", "dirty", or "unknown". Interested in that one?

Comment (calavera) [inline octokit/pull_requests.go]: I didn't know about that one tbh. I'll read about it, sounds interesting for my use case. I guess I could add both in this PR.

**Comment by calavera** [comment]
I've added `mergeable_state` as a string fiend and constant definitions with the values.

@pengwynn maybe it would be worth explaining the different states in the documentation, unfortunately I couldn't find anything about this fiend in the api docs.

**Comment by calavera** [comment]
I don't have any idea why Travis failed either :disappointed:

**Comment by pengwynn** [comment]
@calavera The changes look good. Not sure what's up with Travis.

**Pull request merged**